Apps/W1/HybridGP/app/src/Migration/Support/PurchaseOrders/GPPOMigrator.codeunit.al

```diff
@@ -164,9 +168,6 @@ codeunit 40108 "GP PO Migrator"
             PurchaseLine."Unit Cost" := GPPOP10110.UNITCOST;
             PurchaseLine.Insert(true);
 
-            if QtyShipped > (GPPOP10110.QTYORDER - GPPOP10110.QTYCANCE) then
```

Comment:
I see this moved, was this was not working here for over receipts? Then update Purchase Receipt in here too?

Reply:
It made sense to me to perform extra processing (over receipt, creating the receipt, etc.) after the line has been created. Moving the code doesn't change functionality.